### PR TITLE
Add literature background to dataloder-image-data and mlp-mnist

### PR DESCRIPTION
**Pull request by lilianabs**
This PR adds literature background to the following:

* mlp_mnist
* dataloader-image-data

Issue # [340](https://github.com/FluxML/model-zoo/issues/340)

### CODE REVIEW

File: tutorials/dataloader/dataloader-image-data.jl [hunk 1]
```diff
@@ -1,48 +1,80 @@
...
+
 function get_data(args)
 
-    # Load the MNIST train and test data from MLDatasets
+    ## Load the MNIST train and test data from MLDatasets
     train_x, train_y = MNIST.traindata(Float32)
     test_x, test_y = MNIST.testdata(Float32)
 
-    # Reshape data to 28x28x1 multi-dimensional array
+    ## Reshape data to 28x28x1 multi-dimensional array
```
Comment (ToucheSir) [inline tutorials/dataloader/dataloader-image-data.jl:34]: Question about the double `##`: are these going to be used for some kind of literate rendered output later?

Comment (lilianabs) [reply tutorials/dataloader/dataloader-image-data.jl:34]: @ToucheSir that's right, we are adding these so we can use Literate to convert the file to markdown and notebooks

**Review by ToucheSir — APPROVED** [review]
Let me know when you want this to be merged!

**Comment by lilianabs** [comment]
@ToucheSir this is ready for merging :)

**Pull request merged**